scripts/release

```diff
@@ -480,12 +480,13 @@ generate_release_apidiff() {
   echo "Errors=$ALL_ERROR_LOG_PATH"
 
   # Run new pathdiff script on each umbrella header in array
-  for path_to_component in $(generate_release_components "$old_commit" | grep -v "private"); do
-    component=$(basename $path_to_component)
+  for umbrella_header_path in $(generate_release_umbrella_headers "$old_commit" | grep -v "private"); do
+    umbrella_header=$(basename "$umbrella_header_path")
+    component=$(echo "$umbrella_header" | cut -d'.' -f1 | sed 's:^Material::')
 
-    echo -n "Diffing $component..."
+    echo -n "Diffing $umbrella_header..."
```

Comment:
Maybe add some nifty quotes to surround the name of the thing we're diffing so people know it's literally a thing they can find?

Reply:
Added the path to the component.